Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -48,13 +50,32 @@ public static class ServiceCollectionExtensions
     {
         private const string DefaultDbConnectionName = "DefaultConnection";
 
-        public static IServiceCollection AddWebServer<TStartup>(this IServiceCollection services)
-            => services
+        public static IServiceCollection AddWebServer<TStartup>(
+            this IServiceCollection services,
+            IConfiguration configuration)
+        {
+            services
                 .AddAutoMapperConfigurations<TStartup>()
                 .AddWebServerServices<TStartup>()
                 .AddOptionsWithValidation<ApplicationConfig>()
                 .AddOptionsWithValidation<HealthCheckConfig>();
 
+            var maxRequestLimit = configuration.GetSectionWithValidation<HttpConfig>().MaxRequestSizeLimit;
+            services.Configure<KestrelServerOptions>(options =>
+            {
+                options.Limits.MaxRequestBodySize = maxRequestLimit;
+            });
+
+            services.Configure<FormOptions>(x =>
+            {
+                x.ValueLengthLimit = maxRequestLimit;
+                x.MultipartBodyLengthLimit = maxRequestLimit;
+                x.MultipartHeadersLengthLimit = maxRequestLimit;
```

Comment:
I don't think we want the same value for all of these. Either extract different settings for each, or use only what you need.
For example the default value for MultipartBodyLenghtLimit is 128MB (https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.features.formoptions.multipartbodylengthlimit?view=aspnetcore-7.0#microsoft-aspnetcore-http-features-formoptions-multipartbodylengthlimit) but here we make it smaller.
The Headers limit should definitely be smaller than 35MB.

Reply:
🤦🏻‍♀️🤦🏻‍♀️🤦🏻‍♀️

Removed all updates besides `ValueLengthLimit`